docbook5-topics/xml/generic-entities.ent

```diff
@@ -0,0 +1,410 @@
+<!-- This file is part of the project https://github.com/openSUSE/doc-kit -->
```

Comment:
I wonder if we should commit this file again. In the AsciiDoc RN kit, we are now just using a file from the parent dir via a normal `../`, https://github.com/openSUSE/doc-kit/blob/main/asciidoc-release-notes.MANIFEST#L4. Maybe we can do that here too?

Reply:
Did not notice this, good idea!